thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java

```diff
@@ -222,12 +230,23 @@ private Response processServerErrorResponse(String type, String operation, Strin
     return Response.serverError().entity(responseMessage).build();
   }
 
+  private Response processBadAuthorizationResponse(String type, String operation, String payload, NotAuthorizedException e) {
+    Map<String, String> responseMessage = new HashMap<>();
+    LOG.warn("Authorization error while {} {} with payload {}", operation, type, payload, e);
+    responseMessage.put(type + "Msg", "Authorization Error!");
+    responseMessage.put(type + "Msg-moreInfo", "Configure owners property in " + type + " config");
+    return Response.status(Response.Status.UNAUTHORIZED).entity(responseMessage).build();
+  }
+
   @POST
   @Path("/create-alert")
   @Produces(MediaType.APPLICATION_JSON)
   @Consumes(MediaType.APPLICATION_JSON)
+  @PermitAll
```

Comment:
Why `PermitAll`?

Reply:
This doesn't really matter. It comes into play when we define security roles.